tests/core/snapshots/graphql-datasource-with-recurssive-input.md_merged.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: tests/core/spec.rs
+expression: formatter
+---
+schema @server @upstream {
+  query: Query
+}
+
+input PostFilter {
+  name: StringHashFilter
+  rec: PostFilter
+}
+
+input StringHashFilter {
+  eq: String
+  in: [String]
+}
+
+type Post {
+  name: String!
+  shortName: String
+}
+
+type Query {
+  posts(filter: PostFilter): Post @expr(body: {name: "A", shortName: "B"})
```

Comment:
Do we even need these integration tests?

Reply:
The same are covered by unit tests. I will drop them